topics/imaging/tutorials/object-tracking-using-cell-profiler/tutorial.md

```diff
@@ -0,0 +1,471 @@
+>   - {% icon param-select %} *"Extract single file"*: `All files`
+>
+>
+{: .hands_on}
+
+## Get the metadata
+
+> ### {% icon comment %} Comment
+>
+> The Starting modules tool combines four CellProfiler modules that are always used together at the start of a pipeline. These modules are:  
+>  * Images
+>  * Metadata
+>  * NamesAndTypes
+>  * Groups
+{: .comment}
+
+> ### {% icon hands_on %} Hands-on: Getting metadata
+>
+> 1. {% tool [Starting modules](toolshed.g2.bx.psu.edu/repos/bgruening/cp_common/cp_common/3.1.9+galaxy1) %} with the following parameters:
+>
+>     - Images
+>        - *"Do you want to filter only the images?"*: `Select the images only`
+>
+>     - Metadata
+>        - *"Do you want to extract the metadata?"*: `Yes, specify metadata`
+>
+>          - {% icon param-repeat %} Insert new metadata
+>            - *"Metadata extraction method"*: `Extract from file/folder names`
+>            - *"Metadata source"*: `File name`
+>            - *"Select the pattern to extract metadata from the file name"*: `field1_field2_field3`
+>            - *"Extract metadata from"*: `Images matching a rule`
+>
+>              - *"Match the following rules"*: `All`
+>              - {% icon param-repeat %} Insert filtering rules:
+>                - *"Select the filtering criteria"*: `File`
+>                - *""operator*: `Does`
+>                - *"contain"*: `Contain regular expression`
+>                - *"match_value"*: `GFPHistone`
+>
+>     - NamesAndTypes
+>       - *"Process 3D"*: `No, do not process 3D data` 
+>       - *"Assign a name to"*: `Give images one of several names depending on the metadata`
+>         - {% icon param-repeat %} Insert another image:
+>           - *"Match the following rules"*: `All`
+>           - {% icon param-repeat %} Insert rule:
+>
+>             - *"Select rule criteria"*: `File`
+>
+>               - *"operator"*: `Does`
+>               - *"contain"*: `Contain regular expression`
+>               - *"match_value"*: `GFPHistone`
+>
+>           - *"Select the image type"*: `Color image`
+>           - *"Name to assign these images"*: `OrigColor` 
+>           - *"Set intensity range from"*: `Image metadata` 
+>
+>       - *"Image matching method"*: `Metadata`
+>
+>     - Groups
+>       - *"Do you want to group your images?"*: `Yes, group the images` 
+>       - *"Metadata category"*: `field1` 
+> 
+>
+{: .hands_on}
+
+> ### {% icon question %} Question
+>
+> How are we capturing metadata and what type of metadata are we getting?
+>
+> > ### {% icon solution %} Solution
+> >
+> > Metadata is obtained from the filenames by extracting three text fields separated by an underscore. The metadata we get is "DrosophilaEmbryo", "GFPHistone" and numbers with leading zeros. These three fields represent respectively the sample identifier, the marker visualized in the image and the index in the time series.
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon question %} Question
```

Comment:
These questions can be merged probably. Just make it a numbered list in the first question, 1. How are we, .. 2 How will...

Reply:
Thanks @hexylena! We are keeping them separate to avoid giving away answers to previous questions or having to think about several unrelated topics at once

Reply:
Interesting! I hadn't seen this done before in the GTN. sounds good!